litestar/constants.py

```diff
@@ -17,26 +18,8 @@
 REDIRECT_STATUS_CODES: Final = {301, 302, 303, 307, 308}
 REDIRECT_ALLOWED_MEDIA_TYPES: Final = {MediaType.TEXT, MediaType.HTML, MediaType.JSON}
 RESERVED_KWARGS: Final = {"state", "headers", "cookies", "request", "socket", "data", "query", "scope", "body"}
+SCOPE_STATE_NAMESPACE: Final = "__litestar__"
 SKIP_VALIDATION_NAMES: Final = {"request", "socket", "scope", "receive", "send"}
 UNDEFINED_SENTINELS: Final = {Signature.empty, Empty, Ellipsis, MISSING, UnsetType}
 WEBSOCKET_CLOSE: Final = "websocket.close"
 WEBSOCKET_DISCONNECT: Final = "websocket.disconnect"
-
-# keys for internal stuff that we store in the "__litestar__" namespace of the scope state
-SCOPE_STATE_NAMESPACE: Final = "__litestar__"
-
-SCOPE_STATE_ACCEPT_KEY: Final = "accept"
-SCOPE_STATE_BASE_URL_KEY: Final = "base_url"
-SCOPE_STATE_BODY_KEY: Final = "body"
-SCOPE_STATE_CONTENT_TYPE_KEY: Final = "content_type"
-SCOPE_STATE_COOKIES_KEY: Final = "cookies"
-SCOPE_STATE_CSRF_TOKEN_KEY: Final = "csrf_token"  # possible hardcoded password
-SCOPE_STATE_DEPENDENCY_CACHE: Final = "dependency_cache"
-SCOPE_STATE_DO_CACHE: Final = "do_cache"
-SCOPE_STATE_FORM_KEY: Final = "form"
-SCOPE_STATE_IS_CACHED: Final = "is_cached"
-SCOPE_STATE_JSON_KEY: Final = "json"
-SCOPE_STATE_MSGPACK_KEY: Final = "msgpack"
-SCOPE_STATE_PARSED_QUERY_KEY: Final = "parsed_query"
-SCOPE_STATE_RESPONSE_COMPRESSED: Final = "response_compressed"
-SCOPE_STATE_URL_KEY: Final = "url"
```

Comment:
I know we have a comment that it's for internal stuff, but would this be considered public API since it's not under a private namespace?

Reply:
Yeah - strictly you are right, but the scope state storage is meant to be implementation detail. I'd prefer to ask for forgiveness and put them back in if someone kicks up a stink about it.

If they are relying on these, the fact that the constants are gone will be the least of their worries if this PR goes through.

Reply:
We treat `utils` like a private namespace. It's always been like that. We recently already had a conversation about this when I removed some unused utils. How about we just make it explicitly private?

Reply:
I think making it explicitly private is a good idea. If we can slowly do that for all private stuff (where we haven't already), then whenever we change stuff there, we don't have to worry about backwards compatibility.

Reply:
This would certainly make it easier to make changes and to know where the line should be between needing to deprecate functionality and being able to remove it.

Of these names in this module, a good chunk of them haven't made it into a release yet - so I will remove them here, and deprecate the others.